### PR TITLE
Create GetVirtualMachineInstanceViewPlannedMaintenance.json

### PR DESCRIPTION
**Pull request by mimckitt**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by mimckitt** [comment]
Trying to add the following sample to this page: https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/instanceview#get-virtual-machine-instance-view.



**API output**
Get instance view of a virtual machine with planned maintenance 

**Sample Request**
`GET https://management.azure.com/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM/instanceView?api-version=2020-06-01`

**Sample Response:**

```
{
  "platformUpdateDomain": 0,
  "platformFaultDomain": 0,
  "computerName": "RS186VM8",
  "osName": "Windows Server 2019 Datacenter",
  "osVersion": "Microsoft Windows NT 10.0.17763.0",
  "vmAgent": {
    "vmAgentVersion": "2.7.41491.993",
    "statuses": [
      {
        "code": "ProvisioningState/succeeded",
        "level": "Info",
        "displayStatus": "Ready",
        "message": "GuestAgent is running and processing the extensions.",
        "time": "2020-09-17T16:02:22-07:00"
      }
    ],
    "extensionHandlers": [
      {
        "type": "Microsoft.Azure.Geneva.GenevaMonitoring",
        "typeHandlerVersion": "2.18.0.2",
        "status": {
          "code": "ProvisioningState/succeeded",
          "level": "Info",
          "displayStatus": "Ready",
          "message": "ExtensionOperation:enable. Status:Success"
        }
      },
      {
        "type": "Microsoft.EnterpriseCloud.Monitoring.MicrosoftMonitoringAgent",
        "typeHandlerVersion": "1.0.18038.1",
        "status": {
          "code": "ProvisioningState/succeeded",
          "level": "Info",
          "displayStatus": "Ready",
          "message": "This virtual machine has successfully connected to Azure Log Analytics."
        }
      }
    ]
  },
  "maintenanceRedeployStatus": {
    "isCustomerInitiatedMaintenanceAllowed": true,
    "preMaintenanceWindowStartTime": "2020-08-31T17:00:00-07:00",
    "preMaintenanceWindowEndTime": "2020-09-30T17:00:00-07:00",
    "maintenanceWindowStartTime": "2020-10-01T17:05:00-07:00",
    "maintenanceWindowEndTime": "2020-10-02T17:00:00-07:00",
    "lastOperationResultCode": "None"
  },
  "disks": [
    {
      "name": "RS186VM8_OsDisk_1_a8ffd146a1a244a79240bcd716769ffa",
      "statuses": [
        {
          "code": "ProvisioningState/succeeded",
          "level": "Info",
          "displayStatus": "Provisioning succeeded",
          "time": "2020-08-06T15:55:17.5559179-07:00"
        }
      ]
    }
  ],
  "bootDiagnostics": {
    "consoleScreenshotBlobUri": "https://rs186canarytestdiag.blob.core.windows.net/bootdiagnostics-rs186vm8-69f5179a-cf72-4c1b-bb04-4278e820096b/RS186VM8.69f5179a-cf72-4c1b-bb04-4278e820096b.screenshot.bmp",
    "serialConsoleLogBlobUri": "https://rs186canarytestdiag.blob.core.windows.net/bootdiagnostics-rs186vm8-69f5179a-cf72-4c1b-bb04-4278e820096b/RS186VM8.69f5179a-cf72-4c1b-bb04-4278e820096b.serialconsole.log"
  },
  "extensions": [
    {
      "name": "Microsoft.Azure.Geneva.GenevaMonitoring",
      "type": "Microsoft.Azure.Geneva.GenevaMonitoring",
      "typeHandlerVersion": "2.18.0.2",
      "statuses": [
        {
          "code": "ProvisioningState/succeeded",
          "level": "Info",
          "displayStatus": "Provisioning succeeded",
          "message": "ExtensionOperation:enable. Status:Success",
          "time": "2020-09-17T16:02:18-07:00"
        }
      ]
    },
    {
      "name": "MicrosoftMonitoringAgent",
      "type": "Microsoft.EnterpriseCloud.Monitoring.MicrosoftMonitoringAgent",
      "typeHandlerVersion": "1.0.18038.1",
      "statuses": [
        {
          "code": "ProvisioningState/succeeded",
          "level": "Info",
          "displayStatus": "Provisioning succeeded",
          "message": "Latest configuration has been applied to the Microsoft Monitoring Agent."
        }
      ]
    }
  ],
  "hyperVGeneration": "V1",
  "statuses": [
    {
      "code": "ProvisioningState/succeeded",
      "level": "Info",
      "displayStatus": "Provisioning succeeded",
      "time": "2020-08-12T11:43:35.8559975-07:00"
    },
    {
      "code": "PowerState/running",
      "level": "Info",
      "displayStatus": "VM running"
    }
  ]
}

```

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @mimckitt could you pls fix CI failure?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json [hunk 1]
```diff
@@ -2922,9 +2922,11 @@
           },
           "Get instance view of a virtual machine placed on a dedicated host group through automatic placement.": {
             "$ref": "./examples/GetVirtualMachineInstanceViewAutoPlacedOnDedicatedHostGroup.json"
-          }
+          },
+		      "Get Virtual Machine Instance View with planned maintenance.": {
+		        "$ref: "./examples/GetVirtualMachineInstanceViewPlannedMaintenance.json"
```
Comment (haagha) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:2927]: "$ref": instead of "$ref:

**Review by haagha — CHANGES_REQUESTED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json (hunk 1 above)
Comment (mimckitt) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json]: thank you!

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/GetVirtualMachineInstanceViewPlannedMaintenance.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      "statuses": [
+        {
+          "code": "ProvisioningState/succeeded",
+          "level": "Info",
+          "displayStatus": "Provisioning succeeded",
+          "message": "Latest configuration has been applied to the Microsoft Monitoring Agent."
+        }
+      ]
+    }
+  ],
```
Comment (haagha) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/GetVirtualMachineInstanceViewPlannedMaintenance.json:70]: are the paranthesis balanced here?

**Pull request closed without merging**